packages/blockchain-link-utils/src/solana.ts

```diff
@@ -0,0 +1,205 @@
+import { ParsedInstruction, ParsedTransactionWithMeta } from '@solana/web3.js';
+import BigNumber from 'bignumber.js';
+import { Target, Transaction } from 'packages/blockchain-link-types/lib';
```

Comment:
this import is incorrect, it should never start with `packages`. please replace `packages` with `@trezor`

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/9498/commits/2ebf2108183d43fae965bafaaee72afb8e5fe8e1